js/ui/menu/ui.menu.js

```diff
@@ -525,7 +526,13 @@ class Menu extends MenuBase {
             },
             onSelectionChanged: this._nestedItemOnSelectionChangedHandler.bind(this),
             onItemClick: this._nestedItemOnItemClickHandler.bind(this),
-            onItemRendered: this.option('onItemRendered'),
+            onItemRendered: (e) => {
+                if(that.option('onItemRendered')) {
```

Comment:
onItemRendered: this._nestedItemOnItemRenderedHandler.bind(this), ?

Reply:
Yes, fixed https://github.com/DevExpress/DevExtreme/pull/13787/commits/a2083f83e1e4988b3a450ed247b21187e9430064